recipe/meta.yaml

```diff
@@ -52,6 +52,7 @@ requirements:
     - gmp
     - texttable >=1.6.2
     - python
+    - igraph >=0.10,<0.11
```

Comment:
`igraph` has a run_exports that knows about its compatibility:
https://github.com/conda-forge/igraph-feedstock/blob/main/recipe/meta.yaml#L18-L19
So we should move this to the `host` section and this will mean `igraph` is automatically also included as a run dependency with the appropriate constraint.

Reply:
@szhorvat, the `run_exports` from `igraph` itself will prevent this.  My suggested approach would be that we restrict which version of `igraph` is allowed at build time and the version at runtime will be restricted to be newer than or equal to that version, and less that the next minor version (because of the `x.x` in the `run_exports`).  If that is not correct, we should fix the `run_exports` in `igraph` itself.

Reply:
> Specify a single compatible version of igraph (not a range!) for each specific version of python-igraph

We could do this but it would require ignoring the `run_exports` from `igraph`, which currently allow newer patch versions of `igraph` at runtime than were used a build time.

Reply:
@szhorvat, I believe I have added this in https://github.com/conda-forge/python-igraph-feedstock/pull/98/commits/1bd16f8b880f876ac3469506c7ebb6740c73b95e.  I used the version associated with the submodule that is used to vendor `igraph` in `python-igraph` 0.11.9.